README.md

```diff
@@ -33,37 +38,30 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
 
 4. Start running your own analysis!
 
-    <!-- TODO nf-core: Update the example "typical command" below used to run the pipeline -->
-
-    ```bash
-    nextflow run nf-core/pgdb -profile <docker/singularity/podman/conda/institute> --input '*_R{1,2}.fastq.gz' --genome GRCh37
-    ```
+```bash
+nextflow run nf-core/pgdb -profile <docker/singularity/podman/conda/institute> --ensembl_name homo_sapines --ensembl false
```

Comment:
The pipeline does not have input files. Basically, it will pull all the data from the major providers ENSEMBL, COSMIC or cBioportal. 

We have added made the default parameters false.